ceph.spec.in

```diff
@@ -131,6 +131,7 @@ BuildRequires:	python-pecan
 BuildRequires:	python-requests
 BuildRequires:	python-virtualenv
 BuildRequires:	python-werkzeug
+BuildRequires:	socat
```

Comment:
@tchaikov I see what you mean, now, and I think it's a good idea. One possible danger, though - people might forget that new unit test dependencies need to be added to ceph.spec.in for RPM, but to install-deps.sh for DEB. It's somewhat non-intuitive.